.github/workflows/build.yml

```
@@ -0,0 +1,29 @@
+        with:
+          version: 3.1.57
+
+      - name: Compile
+        run: |
+          cd lib/src
+          make clean
+          make astro
+          make install
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)